web/src/js/components/common/HideInStatic.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+export default function HideInStatic({className, children }) {
+    return MITMWEB_STATIC ? null : ( <div className={className}>{children}</div> )
```

Comment:
Shall we try to update to React 16 beta right away so that we do not have to use the wrapper div here? This would allow us to do
```jsx
<HideInStatic>
    <p>foo</p>
    <p>bar</p>
</HideInStatic>
```

See https://github.com/facebook/react/issues/3415, https://github.com/facebook/react/issues/2127, https://github.com/facebook/react/issues/10294